src/OnePlusBot/Base/CommandHandler.cs

```diff
@@ -34,8 +35,18 @@ public async Task InstallCommandsAsync()
         public async Task RoleReact(IUserMessage message)
         {
             Console.WriteLine("RoleReact called.");
-
             Global.RoleManagerId = message.Id;
+            try
```

Comment:
The message ID will never be null and failing to write will not likely happen, try catch not needed.